RHEL/7/input/profiles/stig-rhel7-server-upstream.xml

```diff
@@ -42,6 +42,8 @@
 <select idref="banner_etc_issue" selected="true" />
 <select idref="sshd_enable_warning_banner" selected="true" />
 <select idref="ftp_present_banner" selected="true" />
+<select idref="dconf_gnome_banner_enabled" selected ="true" />
+<select idref="dconf_login_banner_text" selected="true" />
```

Comment:
@shawndwells This should be `<select idref="dconf_gnome_login_banner_text" selected="true" />`